ouroboros-network/lib/Ouroboros/Network/PeerSelection/Governor/Types.hs

```diff
@@ -1179,6 +1179,8 @@ peerSelectionStateToView
         inProgressDemoteHot
       }
     =
+    -- TODO: investigate whether the demotion sets can include the async demotions
+    -- as well
```

Comment:
Is this a left over, or something that deserves an issue?

Reply:
Could you create an issue for it, and add the issue number in the `TODO` comment.